draft-ietf-quic-transport.md

```diff
@@ -374,27 +398,18 @@ Check the flags in the common header
 
 ## Regular Packets
 
-Each Regular packet's header consists of a Common Header followed by fields
-specific to Regular packets, as shown below:
+Each Regular packet's payload consists of fields shown below:
```

Comment:
I would change this to: "Each Regular packet contains additional header fields followed by payload, as shown below:"